app/javascript/elm/src/Data/SelectedSession.elm

```diff
@@ -122,23 +129,40 @@ view session heatMapThresholds linkIcon toMsg =
         [ p [ class "single-session-name" ] [ text session.title ]
         , p [ class "single-session-username" ] [ text session.username ]
         , p [ class "single-session-sensor" ] [ text session.sensorName ]
-        , div []
-            [ div [ class "single-session-avg-color", class <| Data.Session.classByValue (Just session.average) heatMapThresholds ] []
-            , span [] [ text "avg. " ]
-            , span [ class "single-session-avg" ] [ text <| String.fromInt <| round session.average ]
-            ]
-        , div [ class "session-numbers-container" ]
-            [ div [ class "single-min-max-container" ]
-                [ div [ class "single-session-color", class <| Data.Session.classByValue (Just session.min) heatMapThresholds ] []
-                , span [] [ text "min. " ]
-                , span [ class "single-session-min" ] [ text <| String.fromFloat session.min ]
-                ]
-            , div [ class "single-min-max-container" ]
-                [ div [ class "single-session-color", class <| Data.Session.classByValue (Just session.max) heatMapThresholds ] []
-                , span [] [ text "max. " ]
-                , span [ class "single-session-max" ] [ text <| String.fromFloat session.max ]
-                ]
-            ]
+        , case session.selectedMeasurements of
+            [] ->
+                div [ class "single-session-placeholder" ] []
```

Comment:
what's this for?

Reply:
I needed an element that I can style, so it's the same size as the div from the other case branch. Otherwise, the neighboring elements were changing positions.